apps/client/src/components/Home/home.tsx

```diff
@@ -1,10 +1,12 @@
 import { LandingScreen } from '../landingScreen';
+import { motion } from 'framer-motion';
+import { ANIMATION_TEMPLATE } from '../../common/constant';
```

Comment:
In future, we can add some aliases so that we no need to relay on the paths relative paths ../../common/constant 

something like @common/constant

Reply:
Yeah, I have a plan for this, I just didn't got time. We just to configure vite for this, definitely we will be doing this